scripts/gen_binding.py

```diff
@@ -427,7 +431,8 @@ def normalize_arg(arg):
 import {{ arrayArg }} from "../ffi/ffi_bind_utils";
 import {{ fl }} from "../ffi/ffi_flashlight";
 import {{ TensorInterface }} from "./tensor_interface";
-export const gen_tensor_op_shim = (wrapFunc: Function) => {{
+import type {{ Tensor }} from "./tensor";
```

Comment:
hmm, this is circular right?

@jacobkahn thoughts?

Reply:
maybe it doesn't matter because this is `import type`, which erases itself at runtime

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html

Reply:
TIL that `import type` has no runtime effects/can't create circular dep issues — nice! :D